packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -74,21 +74,21 @@ class ChartController extends DisposableController
   double _tickWidth = 10.0;
 
   /// Number of ticks visible (on the X-axis);
-  int visibleTicks;
+  int? visibleTicks;
```

Comment:
can this be late?

Reply:
I do not think so, because there is getter numberOfVisibleXAxisTicks that can be invoked before initialization.